### PR TITLE
Attempt to re-enable the commented out CI steps

### PR DESCRIPTION
**Pull request by palnabarun**
- [x] Upgrade Postgres in CI environment to 12.2
- [x] Upgrade coverage to 5.0.4
- [x] Re-enable tests on Python 3.7

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # System information
 os: linux
 addons:
-  postgresql: "9.3"
+  postgresql: "12.2"
```
Comment (ananyo2012) [inline .travis.yml]: Does psycopg2-binary needs this specific postgresql version or this is just for using the latest ?

Comment (palnabarun) [reply .travis.yml]: psycopg2 works for Postgres 7.4 to Postgres 12.

Also, why pscopg2-binary and not psycopg2? Ans: https://www.postgresql.org/message-id/CA%2Bmi_8bd6kJHLTGkuyHSnqcgDrJ1uHgQWvXCKQFD3tPQBUa2Bw%40mail.gmail.com

**Comment by pradyunsg** [comment]
Yay!

**Comment by palnabarun** [comment]
Thanks @ananyo2012 @pradyunsg for reviewing the changes.

**Pull request merged**